meteor/server/api/deviceTriggers/RundownsObserver.ts

```diff
@@ -0,0 +1,66 @@
+import { Meteor } from 'meteor/meteor'
+import { RundownId, RundownPlaylistId } from '@sofie-automation/corelib/dist/dataModel/Ids'
+import _ from 'underscore'
+import { Rundowns } from '../../../lib/collections/Rundowns'
+
+const REACTIVITY_DEBOUNCE = 20
+
+type ChangedHandler = (rundownIds: RundownId[]) => () => void
+
+export class RundownsObserver {
+	#rundownsLiveQuery: Meteor.LiveQueryHandle
+	#rundownIds: Set<RundownId> = new Set<RundownId>()
+	#changed: ChangedHandler | undefined
+	#cleanup: (() => void) | undefined
+
+	constructor(activePlaylistId: RundownPlaylistId, onChanged: ChangedHandler) {
+		this.#changed = onChanged
+		const cursor = Rundowns.find(
+			{
+				playlistId: activePlaylistId,
+			},
+			{
+				projection: {
+					_id: 1,
+				},
+			}
+		)
+		this.#rundownsLiveQuery = cursor.observe({
+			added: (doc) => {
+				this.#rundownIds.add(doc._id)
+				this.updateRundownContent()
+			},
+			changed: (doc) => {
+				this.#rundownIds.add(doc._id)
+				this.updateRundownContent()
```

Comment:
I don't think this `changed` will ever be fired, I think I checked and a document starting/stopping matching the query is an added/removed.  
So this will only catch the change of the `projection` fields. Which in this case is nothing (_id is immutable)

But if it did fire, should it be blindly calling `this.updateRundownContent()`, or only when it added something to this.#rundownIds?